src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -5027,6 +5078,211 @@ export class PostgresCohortMembersService {
 
+  /**
+   * Processes date filter condition for createdAt or updatedAt fields
+   * Handles validation, logging, and error handling consistently across all methods
+   *
+   * @param fieldName - The field name ('createdAt' or 'updatedAt')
+   * @param value - The date/time filter value
+   * @param apiId - Optional API ID for logging context
+   * @returns SQL condition string or null if invalid
+   */
+  private processDateFilterCondition(
+    fieldName: 'createdAt' | 'updatedAt',
```

ESLint

[error] Replace `'createdAt'·|·'updatedAt'` with `"createdAt"·|·"updatedAt"`

(prettier/prettier)